model/intent/record_eeg.py

```diff
@@ -0,0 +1,92 @@
+import argparse
+import time
+import pickle
+
+from brainflow.board_shim import BoardShim, BrainFlowInputParams, BoardIds
+
+window_seconds = 10
+
+def main():
+    parser = argparse.ArgumentParser()
+    # use docs to check which parameters are required for specific board, e.g. for Cyton - set serial port
+    parser.add_argument('--timeout', type=int, help='timeout for device discovery or connection', required=False,
+                        default=0)
+    parser.add_argument('--ip-port', type=int, help='ip port', required=False, default=0)
+    parser.add_argument('--ip-protocol', type=int, help='ip protocol, check IpProtocolType enum', required=False,
+                        default=0)
+    parser.add_argument('--ip-address', type=str, help='ip address', required=False, default='')
+    parser.add_argument('--serial-port', type=str, help='serial port', required=False, default='')
+    parser.add_argument('--mac-address', type=str, help='mac address', required=False, default='')
+    parser.add_argument('--other-info', type=str, help='other info', required=False, default='')
+    parser.add_argument('--serial-number', type=str, help='serial number', required=False, default='')
+    parser.add_argument('--board-id', type=int, help='board id, check docs to get a list of supported boards',
+                        required=True)
+    parser.add_argument('--file', type=str, help='file', required=False, default='')
+    parser.add_argument('--actions', type=int, help='number of actions to record',
+                        required=True)
+    parser.add_argument('--sessions', type=int, help='number of sessions per action to record',
+                    required=False, default=2)
+    # board id by name or id
+    parser.add_argument('--board-id', type=str, help='board id or name, check docs to get a list of supported boards',
```

Comment:
Having issues running this since `--board-id` is defined twice.  Should this be `--master_board`?

Reply:
what's the specific error? I added the board name functionality from main to record_eeg.py so it can take either ids or names now

Reply:
oh woops I didn't hve my glasses on, yeah its being defined twice let me fix that

Reply:
Is it still not working? Turns out I was defining it twice since I was copying the name or id functionality from main, so I put in a commit recently to fix that.

Reply:
recent changes fixed it, thanks!